pkg/status/health/health_test.go

```diff
@@ -83,31 +83,13 @@ func TestDeregisterBadToken(t *testing.T) {
 	assert.Contains(t, cat.components, token1)
 }
 
-func TestGetHealthy(t *testing.T) {
+func TestUnhealthyAndBack(t *testing.T) {
```

Comment:
I think it's strange to send something on the channel to force the healthiness
https://github.com/DataDog/datadog-agent/pull/2359/files#diff-d957dbeae5abef86a230c313357e1249R104
Also, you might have more details on this than I do, but why do we loop 10 times ? It feels like 2 times are sufficient.

Reply:
test reworked